README.md

```diff
@@ -1,75 +1,89 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# To-do List
 
+> In this project, I will build a simple HTML list of To Do tasks. "To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. This web page will be built using webpack and served by a webpack dev server.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- HTML and CSS
+- Frameworks: N/A
+- Technologies used: GIT, GITHUB, LINTERS
+- npm
+- Webpack
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo
 
+Check **Live Demo** [here.](https://sonickmumba.github.io/To-do-list/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+> To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+- npm (node package manager) – to be able to install packages locally.
+- A github Account – to be able to use github actions.
+
 ### Setup
 
+- Open the terminal.
+- Run cd \path\to\where\you\want\your\local\copy
+- Clone the repo
+   i. Run git@github.com:Sonickmumba/My-Portfolio.git
```

Comment:
[OPTIONAL] Please update the repo path with the proper one (To-do-list instead of My-Portfolio).